addons/background-blur/userstyle.css

```diff
@@ -0,0 +1,22 @@
+.sa-blur-overlay {
+  top: 0;
+  position: fixed;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  backdrop-filter: blur(0px);
+  z-index: 500;
+  pointer-events: none;
+  transition-property: backdrop-filter, background-color;
+  transition-duration: 200ms;
```

Comment:
Set the transition-duration to 100ms, that'd make it feel it's appearing faster.

Reply:
But you'll need to use Mutation Observers too

Reply:
I stand by 200ms being the ideal speed.

Reply:
> I stand by 200ms being the ideal speed.

k, also use easing to make it feel smoother

Reply:
DM me a list of your suggested changes, I'll think about doing them tmr